arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx

```diff
@@ -0,0 +1,254 @@
+---
+title: "How to use Oracles in Arbitrum"
+sidebar_label: Use Oracles in Arbitrum
+description: Learn how to integrate Oracles into your dapp
+reader-audience: developers who want to build on Ethereum/Arbitrum
+content-type: how-to
+---
+
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+In this how-to, you’ll learn how to incorporate popular Oracle services into your Arbitrum dApps. Specifically, each section will provide step-by-step instructions on how to obtain current asset prices from oracles, helping you seamlessly integrate these services.
```

Comment:
@TucksonDev this is one of my favorite docs PRs ever, for so many reasons:

 - Each of the new docs is clearly a specific type of doc
 - Each doc is clearly aligned with structure conventions for its content-type
 - Clear metadata with a description that will catch search queries
 - We set expectations upfront, with an invitation to provide feedback -> clear CTA
 - Headers are sentence-cased, per conventions

All of that to say, the feedback in this review all falls under the category of "nits" / long-tail optimizations. Love love love the thought put into this reader experience 🙏

Reply:
Thanks very much! 😃 
I have to say I learned from the best though. Thank _you_ for your constant guidance and support while creating all this content  ❤️